pxf/settings.gradle

```diff
@@ -19,9 +19,10 @@
 
 rootProject.name = 'pxf'
 
+include 'pxf'
```

Comment:
As pxf has dependencies on 'pxf-hdfs', 'pxf-hive', 'pxf-hbase', 'pxf-json' could we omit them in settings?